Userland/BuggieBox/main.cpp

```diff
@@ -94,6 +98,22 @@ static constexpr Runner s_runners[] = {
 #undef ALIAS_RUNNER_ENTRY
 };
 
+ErrorOr<int> symlinks_main(Main::Arguments arguments)
+{
+    if (arguments.argc < 2) {
+        outln(stderr, "error: Creating symlinks require a path to the BuggieBox binary.");
+        return 1;
+    }
+
+    auto buggiebox_binary_path = arguments.strings[1];
+    for (auto& runner : s_runners) {
+        auto path = TRY(String::formatted("/bin/{}", runner.name));
+        if (Core::System::symlink(buggiebox_binary_path, path.bytes_as_string_view()).is_error())
+            outln(stderr, "warning: Failed to create symlink at {} to {}.", path, buggiebox_binary_path);
+    }
+    return 0;
+}
```

Comment:
I'd argue that this is not really BuggieBox' duty. If we can call BuggieBox to run `symlinks`, we can just use the commands directly.